usage/xbps/index.markdown

```diff
@@ -95,12 +100,12 @@ After installing any of them don't forget to synchronize the repository data:
 
 Static binaries for Linux built with the *musl C library* are available at:
 
-- [aarch64](http://repo.voidlinux.eu/static/xbps-static-latest.aarch64-musl.tar.xz)
-- [armv6hf](http://repo.voidlinux.eu/static/xbps-static-latest.armv6l-musl.tar.xz)
-- [i686](http://repo.voidlinux.eu/static/xbps-static-latest.i686-musl.tar.xz)
-- [x86\_64](http://repo.voidlinux.eu/static/xbps-static-latest.x86_64-musl.tar.xz)
-- [mipsel32](http://repo.voidlinux.eu/static/xbps-static-latest.mipsel-musl.tar.xz)
-- [mips32](http://repo.voidlinux.eu/static/xbps-static-latest.mips-musl.tar.xz)
+- [aarch64](http://alpha.us.repo.voidlinux.org/static/xbps-static-latest.aarch64-musl.tar.xz)
```

Comment:
These should all be alpha.de to be consistent.

Reply:
Fixed.